user/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls import path
-from .views import Userview, Loginview
+from .views import UserDetailsView, Userview, Loginview
 
 urlpatterns = [
-    path('users/', Userview.as_view()),
-    path('login/', Loginview.as_view())
-    
+    path('user/', Userview.as_view()),
+    path('login/', Loginview.as_view()),
+    path('userdetails/', UserDetailsView.as_view())
```

Comment:
No need to create endpoint with separate name. Just send the id in the url.

Reply:
fixed